features/environment.py

```diff
@@ -275,6 +277,14 @@ def __init__(
             job_suffix = job_suffix.split("PR-")[-1]
         timed_job_tag += str(job_suffix)
```

Comment:
I recall there was one cloud in the past that truncated tags or outright rejection when trying to use tags > some length. Google searching it looks like ec2 is 256 char, azure 512 char and gcp is 63 character limit. So, I think we have to be aware of those limits when defining our unique tags. I don't **think** this extra 8 characters will bump up against those limits. But, if it does we can trim/compress/redact where necessary

Reply:
With the added 9 characters, we have 56 - so we should be good even for the lower gcp limit.